src/Jeffijoe.MessageFormat.MetadataGenerator/Plural/Parsing/RuleParser.cs

```diff
@@ -198,6 +199,8 @@ private IRightOperand[] ParseRightOperand()
 
         private OrCondition ParseOrCondition()
         {
+            var andWordSpan = "and".AsSpan();
```

Comment:
I don't know if this actually makes a difference but you can make it a `private const string` at the class level.

Reply:
Looks the same to me
https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0ATEBqAPgAQCYBGAWACgL8BmAAiNoGUAHAQwDsBnCgbwtoG1mUAJYA3VgBcYtSF0n1iABlocMtALy0ARGu0BufoOHipMkewUBhWGYCC7DAFVOMKxHmNJo9gHMAFACURgJ85IIRtBJQtJxs7JqqjgB0dpwsHEGG4ZEC+ADssfHJADIwfpIAFtkRAL4UIfR0FgoAIhBeAK4AZt0A6iJV7p7eFgHBOaGNEdGxnQC2iUrZ04Ld0LT+syJL+rQ7ADy0ysv72NgTuVOTVwKcC7TYWjYw9o4ubh6ckl4+4zVXeqUG4RApzeYAgRAxomCTSfaWWgvN7OVwASXYABsLDBfmMgo0wrdZnEOIldI5tKl0vEsqs8oVSexSuVfFVIbRoTcaAi2h1JD1+oNKhjsexcaM/ASbkSriSHlplg0QWsNltWDEdoq9odjkpTiJzpcrrLbncHk8kbZpA5UTBRTi8VLAhy6srbmD7hDGkDakA

Reply:
Nice! Thanks for verifying!